examples/end-to-end-examples/sec_10k_qa/mcli-yamls/06a_deploy_llm.yaml

```diff
@@ -1,35 +1,44 @@
 name: mpt-7b-finetuned
 
 # Deployment configuration
-gpu_num: 1  # Number of GPUs to use
-# cluster: r0z0 # Name of the cluster to use for this run
-gpu_type: a10 # Type of GPU to use.
+compute:
+  gpus: 1  # Number of GPUs to use
```

Comment:
Can you leave a comment here of the resource requirements to deploy the 30b?

Reply:
Can do. But I believe it might not work with 30B. Would downloader work if there are multiple pytorch_model.bin files? https://github.com/mosaicml/examples/blob/main/examples/end-to-end-examples/sec_10k_qa/deployment_download_helper.py#L32

Reply:
I believe if the conversion steps in this example are followed you will end up with just one large bin file.

Reply:
Will update the gpus in download_and_convert as well.